apps/system/lockscreen/js/lockscreen_notifications.js

```diff
@@ -579,26 +579,16 @@
   LockScreenNotifications.prototype.updateTimestamps =
   function lsn_updateTimestamps() {
     var timestamps = [...document.querySelectorAll('.notification .timestamp')];
+    var formatter = navigator.mozL10n.DateTimeFormat();
+
     timestamps.forEach((element) => {
-      element.textContent =
-        this.prettyDate(new Date(element.dataset.timestamp));
+      var date = new Date(element.dataset.timestamp);
+      formatter.relativeDate(date, true).then(str => {
+        element.textContent = str;
+      });
     });
   };
 
-  /**
-   * Display a human-readable relative timestamp.
-   */
-  LockScreenNotifications.prototype.prettyDate =
```

Comment:
There is another one like this in notification_screen.js. Should we remove it as well?